<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signalGeneratorMainWindow</class>
 <widget class="QMainWindow" name="signalGeneratorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Url:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Export format:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>http://</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="btnCalculateSignal">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Calculate signal</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>701</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="signalTypeComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>sum_int_signal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>int_page_signal_256</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ord_int_signal</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="exportFormatComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>xlsx</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>csv</string>
     </property>
    </item>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>700</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="actionResultabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
